### PR TITLE
feat:[OPS-23397]: dlite chart change for making it work in QA

### PR DESCRIPTION
**Pull request by chandra1249**
# Commit Checklist

Thank you for creating a pull request! To help us review / merge this can you make sure that your PR adheres as much as possible to the following.

## The Basics

If you are adding new functionality, please provide evidence of the new functionality.

### CODE REVIEW

File: chart/templates/config.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ metadata:
   annotations: {{- include "harnesscommon.tplvalues.render" ( dict "value" .Values.global.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 data:
+ {{- include "harnesscommon.tplvalues.render" ( dict "value" .Values.config "context" $) | nindent 2}}
   pool: |-
```
Comment (arya-harness) [inline chart/templates/config.yaml:15]: better to put the mount file in another configmap, so files are not mounted as pool var is not mounted as envVar

File: chart/templates/deployment.yaml [hunk 2]
```diff
@@ -69,29 +69,14 @@ spec:
           requests:
             memory: 2Gi
             cpu: 1
+        envFrom:
+        - configMapRef:
```
Comment (arya-harness) [inline chart/templates/deployment.yaml:73]: this is nice change, but make sure file pool.yml is in different configmap,

**Review by arya-harness — APPROVED** [review]

File: chart/templates/config.yaml (hunk 1 above)
Comment (arya-harness) [reply chart/templates/config.yaml]: resolved.

File: chart/templates/deployment.yaml (hunk 2 above)
Comment (arya-harness) [reply chart/templates/deployment.yaml:73]: resolved.

File: chart/values.yaml [hunk 3]
```diff
@@ -118,13 +118,15 @@ waitForInitContainer:
...
-  pollIntervalMs: "1000"
-  parallelWorkers: "10000"
+config:
+  DLITE_NAME: dlite-linux
+  DLITE_MANAGER_ENDPOINT: "http://harness-manager:9090"
+  DLITE_POLL_INTERVAL_MILLISECS: "1000"
+  DLITE_PARALLEL_WORKERS: "10000"
+  SKIP_PREPARE_SERVER: "true"
+  DRONE_TRACE: "true"
+  DRONE_DISTRIBUTED_ENABLED: "true"
```
Comment (raghavharness) [inline chart/values.yaml:128]: You can remove DRONE_DISTRIBUTED_ENABLED property, it's not required anymore. You can rebase

**Review by raghavharness — APPROVED** [review]

**Review by abhay084 — APPROVED** [review]

**Pull request merged**